src/integration-test/java/com/commercetools/sync/integration/services/impl/UnresolvedReferencesServiceImplIT.java

```diff
@@ -92,6 +93,44 @@ void save_WithoutException_createsNewCustomObject() {
         assertThat(createdCustomObject.getValue()).isEqualTo(productDraftWithUnresolvedRefs);
     }
 
+    @Test
+    void save_ExistingProductDraftWithoutException_overwriteOldCustomObjectValue() {
+        // preparation
+        final ProductDraft productDraft =
+            SphereJsonUtils.readObjectFromResource(PRODUCT_KEY_1_RESOURCE_PATH, ProductDraft.class);
+
+        final WaitingToBeResolved productDraftWithUnresolvedRefs =
+                new WaitingToBeResolved(productDraft, asSet("foo", "bar"));
+
+        unresolvedReferencesService
+                .save(productDraftWithUnresolvedRefs)
+                .toCompletableFuture()
+                .join();
+
+        final WaitingToBeResolved productDraftWithUnresolvedNewRefs =
+                new WaitingToBeResolved(productDraft, asSet("foo123", "bar123"));
```

Comment:
Because this test case is to verify the replacing the value of an existing custom object value. This `save_WithoutException_createsNewCustomObject()` particular test case assert the custom object creation for the first time.